industry_benchmarks/utils/tests/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
+
+for file in network_setup/transformations/*.json; do
+  relpath=${file:14}  # strip off "network_setup/"
+  dirpath=${relpath%.*}  # strip off final ".json"
+  jobpath="network_setup/${dirpath}.job"
+  for repeat in {0..2}; do
+    cmd="openfe quickrun $file -o results_${repeat}/$relpath -d results_${repeat}/$dirpath"
```

Comment:
Before we run our command, lets check if the output exists first (should be easy to test by running it twice and see if it skips the second time)
```suggestion
    cmd="openfe quickrun $file -o results_${repeat}/$relpath -d results_${repeat}/$dirpath"
    if [ -f results_${repeat}/${relpath}.json ]; then
		echo skipping ${relpath} because result exits
		continue
	fi
```